book/website/community-handbook/consistency/consistency-language.md

```diff
@@ -0,0 +1,46 @@
+(ch-consistency-language)=
+# Language
+
+Language is concerned with the style with which _The Turing Way_ is written.
```

Comment:
This sentence is a bit vague. You could say everything we've covered in the style guide and consistency chapter has something to do with 'style' and the way _The Turing Way_ is written, right?

Reply:
How about 'Language is concerned with the way concepts and ideas in _The Turing Way's_ are expressed.'